docusaurus/docs/cloud/projects/settings.md

```diff
@@ -31,16 +31,36 @@ The *General* tab enables you to check and update the following options for the
   }}
 />
 
-### Modifying git repository branch
+### Modifying git repository & branch
 
-The GitHub or Gitlab repository branch and base directory for a Strapi Cloud project are by default chosen at the creation of the project (see [Creating a project](/cloud/getting-started/deployment)). Both can afterwards be edited via the project's settings.
+The GitHub or Gitlab repository, branch and base directory for a Strapi Cloud project are by default chosen at the creation of the project (see [Creating a project](/cloud/getting-started/deployment)). After the project's creation, via the project's settings, it is possible to:
 
-1. In the *Connected git repository* section of the *General* tab, click on the **Edit** button.
-2. In the *Edit Git settings* dialog, edit the available options of your choice:
+- update the project's repository or switch to another git provider (see [Updating repository](#updating-repository)),
+- edit the project's branch, base directory and deploy on push setting (see [Editing branch](#editing-branch)).
+
+#### Updating repository
+
+1. In the *Connected git repository* section of the *General* tab, click on the **Update repository** button.
+2. (optional) If you wish to not only update the repository but switch to another git provider, click on the **Switch to GitHub/GitLab** button at the bottom of the *Update repository* dialog. You will be redirected to the chosen git provider's authorization settings before getting back to the *Update repository dialog*.
+3. In the *Update repository* dialog, fill in the 3 available settings:
+
+    | Setting name    | Instructions                                                             |
+    | --------------- | ------------------------------------------------------------------------ |
+    | Account         | Choose an account from the drop-down list.                               |
+    | Repository      | Choose a repository from the drop-down list.                             |
+    | Git branch      | Choose a branch from the drop-down list.                                 |
+
+4. Click on the **Save** button.
+5. In the confirmation dialog, confirm your changes by clicking on the **Relink repository** button.
+
```

Comment:
Can we add a warning about the potential destructive nature of such an action if the new repo does not correspond to the app already deployed?